src/pages/ListPage/ListPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ItemsList } from '../../components/ItemsList/ItemsList';
+
+type Props = {
+  items: string[],
+};
+
+export const ListPage: React.FC<Props> = ({ items }) => {
+  return (
+    <ItemsList
+      items={items}
+    />
+  );
+};
```

Comment:
This component doesn't serve any purpose, delete it and use `ItemsList` in its place